docs/UserGuide.md

```diff
@@ -213,4 +246,6 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Sort** | `sort`<br> e.g., `sort appointment asc deadline`
```

Comment:
Should there be 2 formats for the sort command as well?

Reply:
Updated already